index.Rmd

```diff
@@ -4,11 +4,11 @@ author: Mia Giannini
 output: html_document
 ---
 
-# Introduction
+# Introduction 
```

Comment:
## I really like this topic! I'm studying Urban planning and we did undergrad together so you know I love any kind of tree experiment!

Reply:
I can't reply anywhere else so another thought I had was maybe including a legend on your leaflet map? Someone suggested that for me and I found it helpful!

I think the variables you're looking at are tough. There's a lot of other factors that could explain the trend you're looking for like race, employed, infrastructure present, etc. It may be a more positive trend than this shows, but could have to do with other factors being more directly related.

Overall great job, and I am looking forward to seeing the final product!